frame/scheduler/src/benchmarking.rs

```diff
@@ -127,8 +126,10 @@ benchmarks! {
 		);
 	}
 
+	// TODO: Make this more complex and flexible so it can be used in automation.
```

Comment:
I think we have the policy that every `TODO` needs to link to an issue, but actually I would write it as `NOTE: This could be used in automation if made more complex and flexible`

Reply:
I think I want to keep TODO, because it really should be fixed. I can open an issue